services/suggest.js

```diff
@@ -146,54 +146,59 @@ async function findSuggestions(githubApiProvider, url) {
 //        - link: target as a string URL
 //    - preview: object (optional)
 //      - style: string
-function setRoutes(allowedOrigin, githubApiProvider, server) {
-  server.ajax.on('suggest/v1', (data, end, ask) => {
+function setRoutes(allowedOrigin, githubApiProvider, app) {
+  app.get('/[$]suggest/v1', (req, res) => {
```

Comment:
The Scoutcamp AJAX routes implicitly started with $. For some reason the dollar needs to be escaped for Express.